packages/wrangler/src/config/validation.ts

```diff
@@ -2230,12 +2231,13 @@ const validateConsumer: ValidatorFn = (diagnostics, field, value, _config) => {
 
 	const options: {
 		key: string;
-		type: "number" | "string";
```

Comment:
I think you can revert this change. I made it originally because I had to support `concurrency_enabled`.

Reply:
Good call, but I'll just leave it, doesn't hurt to have support for booleans here.